Onboarding/Onboarding/Pages/ProfilePages/Education.cs

```diff
@@ -12,141 +12,167 @@ namespace Onboarding.Pages.ProfilePages
 {
     public class Education
     {
-        public void AddEducation(IWebDriver driver, string country, string university, string title, string degree, string graduationYear)
+        public IWebDriver driver;
+        Education EducationObj;
+        public Education(IWebDriver _driver)
+        {
+            this.driver = _driver;
+        }
+
+        public void EducationStepDefinitions()
```

Comment:
This I believe is not needed is there any reason you are doing this?

Reply:
redundant